var/spack/repos/builtin/packages/py-python3-xlib/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPython3Xlib(PythonPackage):
+    """python3-xlib is python3 version of python-xlib."""
+
+    homepage = "https://www.example.com"
```

Comment:
```suggestion
    homepage = "https://github.com/LiuLang/python3-xlib"
```

Reply:
Oops... Though that URL ends up 404ing... I'll put the pypi one in for now

Reply:
In that case I would just remove the `homepage` variable, the `pypi` attribute will set it automatically.